packages/web-components/src/appointment-bookings/client/components/date-cell.svelte

```diff
@@ -0,0 +1,57 @@
+<script>
+  import TimeCell from './time-cell.svelte'
+  import dayjs from 'dayjs'
+
+  export let date
+
+  // generate dummy meeting slots between 00-00 -> 10-00 - duration - on hours
+  export const mockedTimes = []
+
+  // if not testing
+  if (typeof window.process === 'undefined') {
+    for (let i = 0; i <= 5; i++) {
+      const randomBool = Math.random() >= 0.5
+
+      if (randomBool) {
+        mockedTimes.push(`${i}${i}:${i}${i}`)
+        continue
+      }
+
+      mockedTimes.push(null)
+    }
+  }
+
+  export const formatHeader = date => {
+    return date.format('ddd MMM DD')
+  }
+</script>
+
+<style>
+  .date-cell-container {
+    flex: 1;
+    margin-right: var(--cellSpacing);
```

Comment:
https://caniuse.com/#search=css%20variables No IE support for CSS variables am afraid